client/components/thank-you/index.tsx

```diff
@@ -160,22 +156,32 @@ export const ThankYou = ( props: ThankYouProps ): JSX.Element => {
 
 					{ showSupportSection && (
 						<ThankYouSectionContainer>
-							<ThankYouHeaderTitle className="thank-you__body-header wp-brand-font">
+							<ThankYouSectionTitle className="thank-you__body-header wp-brand-font">
 								{ translate( 'How can we help?' ) }
-							</ThankYouHeaderTitle>
-							<p>
+							</ThankYouSectionTitle>
+							<p className="thank-you__help-title">
```

Comment:
Nit: I think it would be better to use `thank-you__help-text` as this isn't strictly speaking a title.